Cogs.Publishers/DataInformation.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Cogs.Publishers
+{
+    public class DataInformation

```

Comment:
We can use the Cogs.Dto.Property class for this, we don't need a new class

https://github.com/Colectica/cogs/blob/master/Cogs.Dto/Property.cs